libraries/AC_AttitudeControl/AC_AttitudeControl_Heli.cpp

```diff
@@ -17,7 +17,7 @@ const AP_Param::GroupInfo AC_AttitudeControl_Heli::var_info[] = {
 
     // @Param: RAT_RLL_P
     // @DisplayName: Roll axis rate controller P gain
-    // @Description: Roll axis rate controller P gain.  Converts the difference between desired roll rate and actual roll rate into a motor speed output
+    // @Description: Roll axis rate controller P gain.  Corrects in proportion to the difference between the desired roll rate vs actual roll rate output
```

Comment:
Corrects in proportion to the difference between the desired roll rate vs actual roll rate

Reply:
copy paste error...fixed